src/speed-dial/index.tsx

```diff
@@ -7,17 +7,37 @@ import * as fabCss from '../theme/default/floating-action-button.m.css';
 import { createICacheMiddleware } from '@dojo/framework/core/middleware/icache';
 import Icon, { IconType } from '../icon';
 
+// Enum used to position the action label
+export enum LabelOrientation {
+	bottom = 'bottom',
+	left = 'left',
+	right = 'right',
+	top = 'top'
+}
+
 export interface ActionProperties {
 	/* On click callback for the action */
 	onClick(): void;
 	/* Title text for the action */
 	title?: string;
+	/* Static label for the action */
+	label?: string;
+	/* Orientation for the label */
+	labelOrientation?: LabelOrientation;
```

Comment:
Note that `label` was intentionally chosen as an independent prop to `title`, the idea being that `title` is meant for a11y-specific titling and may be different than visual helper text (e.g. `label` could say "Click here!" while the `title` is "Click here for deals" for screen readers. If we want to instead consolidate these, that's easy to do as well.